### DISCUSSION TITLE
Custom Portals

### DISCUSSION THREAD

**Discussion by SageSphinx63920** [PurpurMC/Purpur#939] [Feature Requests]
Maybe it's possible to implement portals in any shape

**Comment by ghost** [comment]
*Allow nether portals to be any shape, not only rectangles/squares

**Reply by SageSphinx63920** [reply]
Maybe sideways too

**Reply by katekerllenevich** [reply]
Sideways I have a feeling isn’t possible because the portal blocks have a vertical shape
